src/testjfxapp/GraphicsOptions.java

```diff
@@ -21,21 +20,19 @@
     Label mainMenuLabel;
 
     //Create an object of the MainMenu and TestJFXApp class so we can use it
-    MainMenu mainMenu = new MainMenu();
-    TestJFXApp tetrisGame = new TestJFXApp(10,20,30, mainMenu);
+    MainMenu mainMenu;
 
-    public static void main(String[] args) {
-        launch(args);
+    public GraphicsOptions(MainMenu m) {
+        mainMenu = m;
     }
 
-    @Override
+    //@Override
     public void start(Stage primaryStage) throws Exception {
 
         window = primaryStage;
 
         closeProgram = new Button("Quit");
-        closeProgram.setOnAction(e -> quitProgram());
-
+        closeProgram.setOnAction(e -> mainMenu.quitProgram());
```

Comment:
We don't really need a button to quit directly from a settings menu, people can just click the window close button, or navigate to the main menu.

Reply:
Yeah true, I'll change it to a 'back' button as it's clearly more useful to go back to the previous screen :D